### PR TITLE
deal with the fact we get no results so host is completely unreachable

### PR DESCRIPTION
**Pull request by gergnz**
Had an example where a host was completely unreachable, so needed to actually deal with that and get 100% packet loss recorded.

### CODE REVIEW

**Comment by lifeofguenter** [comment]
curious: how/where are you using this?

File: pingaz/pinger.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def ping(hosts):
     results = {}
     for p in procs:
         out, err = p['proc'].communicate(timeout=30)
-        if p['proc'].returncode > 0:
+        if p['proc'].returncode > 1:
```
Comment (lifeofguenter) [inline pingaz/pinger.py:42]: why this change?

**Comment by gergnz** [comment]
> curious: how/where are you using this?

My use case was a custom site to site VPN configured on an EC2 instance that has recently suffered some downtime. So I wanted to monitor it.

File: pingaz/pinger.py (hunk 1 above)
Comment (gergnz) [reply pingaz/pinger.py:42]: From the fping man page:

```
Exit status is 0 if all the hosts are reachable, 1 if some hosts were unreachable, 
```

**Review by lifeofguenter — APPROVED** [review]
